selfdrive/ui/ui.cc

```diff
@@ -938,7 +945,11 @@ int is_leon() {
   return strstr(str, "letv") != NULL;
 }
 
-
+#ifdef QCOM
```

Comment:
can we handle the persistent params path in `params.cc`? Maybe pass in a flag to `read_db_value` to tell if it's a persistent param. It's all in C++, so you can add an argument with a default value, so the other code using it wouldn't need to change.